luigi/contrib/ecs.py

```diff
@@ -0,0 +1,162 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright 2015 Outlier Bio, LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+"""
+EC2 Container Service wrapper for Luigi
+
+From the AWS website,
+
+> Amazon EC2 Container Service (ECS) is a highly scalable, high performance
+> container management service that supports Docker containers and allows you
+> to easily run applications on a managed cluster of Amazon EC2 instances.
+
+To use ECS, you create a "task definition" JSON that defines the `docker run`
+command for one or more containers in a task or service, and then submit this
+JSON to the API to run the task.
+
+This boto3-powered wrapper allows you to create Luigi Tasks to submit ECS
+`taskDefinition`s. You can either pass a dict (mapping directly to the
+`taskDefinition` JSON) OR an Amazon Resource Name (arn) for a previously
+registered `taskDefinition`.
+
+Requires:
+* boto3 package
+* Amazon AWS credentials discoverable by boto3 (e.g., by using `aws configure`
```

Comment:
Have you checked that the docs render properly? You can use `tox` to generate docs too: https://github.com/spotify/luigi/blob/master/CONTRIBUTING.rst#writing-documentation,